buildSrc/src/main/minimumRuntime/org/elasticsearch/gradle/LoggedExec.java

```diff
@@ -45,11 +83,15 @@ public LoggedExec() {
                             )
                         );
                     }
-                }
-            );
+                });
+            });
         }
     }
 
+    public void setUseFileBuffer(boolean useFileBuffer) {
```

Comment:
I think the fact that this affects log output should be apparent here. Perhaps something like `spoolConsoleOutput`?